api/database/studentsDOM.py

```diff
@@ -153,6 +155,44 @@ def addNewFormId(id, newFormId):
     writeR = dict(mongo.db.students.update({'_id': id}, {'$set': {'form_ids': oldForms}}))
     return writeR['nModified'] > 0
 
+def addNewFile(id, newFileId,newFileName):
+    contents = list(mongo.db.students.find({'_id': id}))
+    if len(contents) != 1:
+        return False
```

Comment:
You return false a few times here, but do you do anything with these failures? Or you would rather use `assert` to raise an `AssertionError`